added two new content under the Oracle page (#167)
rsksmart/devportal: docs/05-dev-tools/oracles/umbrellanetwork.md

---
sidebar_position: 2
sidebar_label: Umbrella Network
title: Umbrella Network

> description: "Umbrella Network provides decentralized and scalable oracles designed for high-performance blockchain applications. " 

Reviewer: ```suggestion
description: "Umbrella Network provides decentralized and scalable oracles designed for high-performance blockchain applications." 
```

> tags: [Umbrella Network , developer tools, rsk, rootstock, ethereum, dApps, smart contracts, testnet]

Reviewer: ```suggestion
tags: [Umbrella Network, developer tools, rsk, rootstock, ethereum, dApps, smart contracts, testnet, Oracles]
```

Reviewer: Include mainnet or testnet in the tag if supported

---

> Umbrella Network provides decentralized and scalable oracles designed for high-performance blockchain applications. 

Reviewer: Include the mainnet or testnet badge in the description. See example for other tools pages.

> The platform combines a decentralized architecture with seamless integration options to deliver reliable data feeds. It is compatible with 12 major blockchains, including Ethereum, BNB Chain, Polygon, and Solana, ensuring extensive support for various blockchain projects.

Reviewer: ```suggestion
The platform combines a decentralized architecture with seamless integration options to deliver reliable data feeds. It is compatible with 12 major blockchains, including Rootstock, Ethereum, BNB Chain, ensuring extensive support for various blockchain projects.
```
